rocAL/rocAL/source/parameters/parameter_rocal_crop.cpp

```diff
@@ -50,36 +50,36 @@ void RocalCropParam::fill_crop_dims() {
     for(uint img_idx = 0; img_idx < batch_size; img_idx++) {
         if (!(_random)) {
             // Evaluating user given crop
-            cropw_arr_val[img_idx] = (crop_w <= in_width[img_idx] && crop_w > 0) ? crop_w : in_width[img_idx];
-            croph_arr_val[img_idx] = (crop_h <= in_height[img_idx] && crop_h > 0) ? crop_h : in_height[img_idx];
+            cropw_arr_val[img_idx] = (crop_w <= in_roi[img_idx].x2 && crop_w > 0) ? crop_w : in_roi[img_idx].x2;
+            croph_arr_val[img_idx] = (crop_h <= in_roi[img_idx].y2 && crop_h > 0) ? crop_h : in_roi[img_idx].y2;
             if (_is_fixed_crop) {
-                x1_arr_val[img_idx] = static_cast<size_t>(std::nearbyintf(_crop_anchor[0] * (in_width[img_idx] - cropw_arr_val[img_idx])));
-                y1_arr_val[img_idx] = static_cast<size_t>(std::nearbyintf(_crop_anchor[1] * (in_height[img_idx] - croph_arr_val[img_idx])));
+                x1_arr_val[img_idx] = static_cast<size_t>(std::nearbyintf(_crop_anchor[0] * (in_roi[img_idx].x2 - cropw_arr_val[img_idx])));
+                y1_arr_val[img_idx] = static_cast<size_t>(std::nearbyintf(_crop_anchor[1] * (in_roi[img_idx].y2 - croph_arr_val[img_idx])));
             } else {
-              x1_arr_val[img_idx] = (x1 >= in_width[img_idx]) ? 0 : x1;
-              y1_arr_val[img_idx] = (y1 >= in_height[img_idx]) ? 0 : y1;
+              x1_arr_val[img_idx] = (x1 >= in_roi[img_idx].x2) ? 0 : x1;
+              y1_arr_val[img_idx] = (y1 >= in_roi[img_idx].y2) ? 0 : y1;
             }
         } else {
           float crop_h_factor_, crop_w_factor_, x_drift, y_drift;
           crop_height_factor->renew();
           crop_h_factor_ = crop_height_factor->get();
           crop_width_factor->renew();
           crop_w_factor_ = crop_width_factor->get();
-          cropw_arr_val[img_idx] = static_cast<size_t> (crop_w_factor_ * in_width[img_idx]);
-          croph_arr_val[img_idx] = static_cast<size_t> (crop_h_factor_ * in_height[img_idx]);
+          cropw_arr_val[img_idx] = static_cast<size_t> (crop_w_factor_ * in_roi[img_idx].x2);
+          croph_arr_val[img_idx] = static_cast<size_t> (crop_h_factor_ * in_roi[img_idx].y2);
```

Comment:
Please remove the space after static_cast<size_t>

Reply:
Done